src/entities/movie/api/utils/createSearchParams.ts

```diff
@@ -0,0 +1,13 @@
+import { GetPaginatedMoviesParams } from "../types";
+
+export const createSearchParams = (params: GetPaginatedMoviesParams): URLSearchParams => {
```

Comment:
I think it's a good candidate for shared folder but with generic type for params